website_sale_suggest_create_account/tests/test_shop_buy.py

```diff
@@ -3,18 +3,24 @@
 import odoo.tests
 
 
-@odoo.tests.tagged("post_install", "-at_install")
+@odoo.tests.tagged("post_installl", "-at_install")
 class TestUi(odoo.tests.HttpCase):
     def test_01_shop_buy(self):
         # Ensure that 'vat' is not empty for compatibility with
         # website_sale_vat_required module
-        portal_user = self.env.ref("base.demo_user0")
-        if not portal_user.partner_id.vat:
-            portal_user.partner_id.vat = "BE1234567"
+        testuser = self.env["res.users"].create(
+            {
+                "email": "testuser@testuser.com",
+                "name": "Test User",
+                "login": "testuser",
+                "password": "testuser",
+            }
+        )
+        if not testuser.partner_id.vat:
+            testuser.partner_id.vat = "BE1234567"
         current_website = self.env["website"].get_current_website()
         current_website.auth_signup_uninvited = "b2b"
         self.env.ref("website_sale_suggest_create_account.cart").active = True
-        self.env.ref(
-            "website_sale_suggest_create_account.short_cart_summary"
-        ).active = True
-        self.start_tour("/shop", "shop_buy_checkout_suggest_account_website")
+        self.start_tour(
+            "/shop", "shop_buy_checkout_suggest_account_website", login="testuser"
```

Comment:
This still makes no sense. You're supposed to test this module without being logged in, which is the point of the test